.github/workflows/ci.yml

```diff
@@ -35,9 +35,14 @@ jobs:
       matrix:
         os: [ ubuntu-latest ]
         python-version: [ '3.8', '3.9', '3.10', '3.11']
+        pydantic-version: [ '1.10.15', '2.7.0' ]
```

Comment:
nit: ` 2.7.1` has just been released (literally minutes ago!). I dont think though that we need to test with the latest versions.

Reply:
✅